invenio_communities/communities/services/config.py

```diff
@@ -105,13 +104,18 @@ class CommunityServiceConfig(RecordServiceConfig):
         "requests": CommunityLink("{+api}/communities/{id}/requests"),
     }
 
+    action_link = CommunityLink("{+api}/communities/{id}/{action_name}",
+                                when=_is_action_available)
+
     links_search = pagination_links("{+api}/communities{?args*}")
     links_featured_search = pagination_links("{+api}/communities/featured{?args*}")
     links_user_search = pagination_links("{+api}/user/communities{?args*}")
     links_community_requests_search = pagination_links(
         "{+api}/communities/{community_id}/requests{?args*}"
     )
 
+    available_actions = [("featured_create", "featured")]
```

Comment:
Lacking context... what is the difference between the 2? 
create new featured and get the featured ones?